shipyard-controller/debug-ui/fetch_events.js

```diff
@@ -0,0 +1,75 @@
+fetchEvents();
+
+function fetchEvents() {
+  const urlParams = new URLSearchParams(window.location.search);
+  const context = urlParams.get("shkeptncontext");
+  const projectname = urlParams.get("projectname");
+
+  let list = document.getElementById("tree_container");
+  list.innerHTML = "";
+
+  fetch(
+    "../sequence/project/" + projectname + "/shkeptncontext/" + context + "/event",
+    {
+      method: "get",
+    }
+  )
+    .then((res) => {
+      return res.json();
+    })
+    .then((response) => {
+      let rootevent_list = document.getElementById("tree_container");
+      let rootevent_li, taskevent_ul, label;
+
+      response.forEach((object) => {
+        if (object.type.split(".").length == 6) {
+          rootevent_li = document.createElement("li");
+
+          label = document.createElement("span");
+          label.innerHTML = object.type;
+
+          rootevent_li.appendChild(label);
+
+          taskevent_ul = document.createElement("ul");
+          taskevent_ul.className = "nested";
+        } else {
+          label.className = "caret";
+
+          let taskevent_li = document.createElement("li");
+
+          let label2 = document.createElement("span");
+          label2.className = "caret";
+          label2.innerHTML = object.type;
+          taskevent_li.appendChild(label2);
+
+          let detail_ul = document.createElement("ul");
+          detail_ul.className = "nested";
+
+          for (let key in object.data) {
+            let li = document.createElement("li");
+            li.innerHTML = key + ": " + object.data[key];
```

Comment:
please, check if it is only TS or is also available in JS 
```suggestion
            li.innerHTML = `${key} :  ${object.data[key]}`;
```